lib/active_admin/view_helpers/display_helper.rb

```diff
@@ -25,7 +25,7 @@ def display_name_method_for(resource)
         @@display_name_methods_cache ||= {}
         @@display_name_methods_cache[resource.class] ||= begin
           methods = active_admin_application.display_name_methods - association_methods_for(resource)
-          method  = methods.detect{ |method| resource.respond_to? method }
+          method  = methods.detect{ |method_name| resource.respond_to? method_name }
```

Comment:
Line is too long. [84/80]<br>Unnecessary spacing detected.<br>Space missing to the left of {.